include/libtorrent/kademlia/node.hpp

```diff
@@ -134,7 +134,7 @@ class TORRENT_EXTRA_EXPORT node : boost::noncopyable
 	int data_size() const { return int(m_storage.num_torrents()); }
 #endif
 
-#ifndef TORRENT_DISABLE_LOGGING
+#if TORRENT_USE_IOSTREAM
 	// TODO: 3 make this print to the DHT logger instead
```

Comment:
sure, maybe it doesn't make sense to print to the log. I'm still not sure about the rationale for this change. USE_IO_STREAM is a macro defining the environment you're building for. This function, which are just used for debugging (in the dht tests), should not be forced upon end-users I think. With this change there's no way to disable it. Arguably, it's not great to have the only way to disable it be by disabling all of logging either, but it's at least a build configuration macro. Ideally it would be moved out to a free function, maybe even living entirely within the tests